images/build/php/install-prereqs.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+# --------------------------------------------------------------------------------------------
+
+set -eux
+
+# prevent Debian's PHP packages from being installed
+# https://github.com/docker-library/php/pull/542
+{
+    echo 'Package: php*';
+    echo 'Pin: release *';
+    echo 'Pin-Priority: -1';
+} > /etc/apt/preferences.d/no-debian-php
+
+# dependencies required for running "phpize"
+# (see persistent deps below)
+PHPIZE_DEPS="autoconf dpkg-dev file g++ gcc libc-dev make pkg-config re2c"
+
+# persistent / runtime deps
+apt-get update && apt-get install -y \
```

Comment:
Its worth checking if using `buildpack-deps:stable` can avoid installing these

Reply:
even if they're already installed in `buildpack-deps:stable`, this command line will return right away (with something like "requirements already satisfied")